src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -509,7 +509,7 @@ static void close_ride_window_for_construction(rct_windownumber number)
 rct_window* window_ride_construction_open()
 {
     ride_id_t rideIndex = _currentRideIndex;
-    close_ride_window_for_construction(rideIndex);
+    close_ride_window_for_construction(EnumValue(rideIndex));
```

Comment:
Probably better to change the signature of `close_ride_window_for_construction()` to take `ride_id_t`?